pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -1448,6 +1448,11 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
                 // Make sure the apiserver can be accessed from WSL through the internal gateway
                 k3sConf.ADDITIONAL_ARGS += ' --tls-san gateway.rancher-desktop.internal';
```

Comment:
That's not actually reachable right?

Reply:
> That's not actually reachable right?

This was not answered. If the address is not reachable, why would we add it to the cert?

Reply:
I was added here: https://github.com/rancher-sandbox/rancher-desktop/pull/6167, so it looks like it is needed because of this: https://github.com/rancher-sandbox/rancher-desktop/issues/6156.

Reply:
Are we using `gateway.rancher-desktop.internal` for WSL `kubeconfig` then?  We're using a symlink again now, so it should be back to `127.0.0.1`?

Reply:
Yeah, it is back to localhost now.

Reply:
I'm still confused: Is `gateway.rancher-desktop.internal` used for anything? Should it still be on the cert?

Because the comment sounds wrong now:

```golang
// Make sure the apiserver can be accessed from WSL through the internal gateway
```